src/screens/MainScreen.tsx

```diff
@@ -14,31 +14,32 @@ type MainScreenProps = NativeStackScreenProps<
   RootStackParams,
   NavigationPages.Issues
 >;
+const enum LinkInputTitles {
```

Comment:
use object in this case. We can introduce localization if you want. Enum is rather used for types, keys